Tangem/App/Services/NotificationManagers/SingleTokenNotifications/TokenNotificationEvent.swift

```diff
@@ -95,15 +100,29 @@ extension TokenNotificationEvent: NotificationEvent {
             )
         case .solanaHighImpact:
             return Localization.warningSolanaFeeMessage
+        case .hasUnfulfilledRequirements(configuration: .missingHederaTokenAssociation(let associationFee)):
+            if let associationFee {
+                return Localization.warningHederaMissingTokenAssociationMessage(
+                    associationFee.formattedValue,
+                    associationFee.currencySymbol
+                )
+            }
+            return Localization.warningHederaMissingTokenAssociationMessageBrief
```

Comment:
как будто через `guard` проще читать 

```suggestion
            guard let associationFee else {
                return Localization.warningHederaMissingTokenAssociationMessageBrief
            }
            
            return Localization.warningHederaMissingTokenAssociationMessage(
                    associationFee.formattedValue,
                    associationFee.currencySymbol
                )
```

Reply:
Поправил